integration-tests/src/node/process_node.rs

```diff
@@ -15,10 +15,11 @@ use nearcore::config::NearConfig;
 use crate::node::Node;
 use crate::user::rpc_user::RpcUser;
 use crate::user::User;
-use actix::{Actor, System};
-use futures::{FutureExt, TryFutureExt};
+use actix::System;
 use near_jsonrpc_client::new_client;
-use near_network::test_utils::WaitOrTimeoutActor;
+use near_network::test_utils::wait_or_timeout;
+
+pub type ControlFlow = std::ops::ControlFlow<()>;
```

Comment:
ditto, I think just a `use` would work, no need for type alias

Reply:
fixed!